### ISSUE TITLE
Webhooks a

### ISSUE THREAD

**Issue by garlik82** [djdembeck/bragibooks#73]
Any chance webhooks can be added? 
Would be nice to use bragibooks to trigger a Plex library scan with [autoscan](https://github.com/Cloudbox/autoscan).

**Comment by djdembeck** [comment]
This sounds promising for sure. I'll try and look at it when AudiobookDB gets closer to release

**Comment by garlik82** [comment]
Thanks :). what's AudiobookDB? a website like imdb or tmdb?

**Comment by djdembeck** [comment]
Precisely. It's an in-development open metadata site specifically designed for audiobooks. Right now that's where my time is dedicated

**Comment by garlik82** [comment]
Uhhh, that's nice. Good luck.
